release-content/0.14/release-notes/13418_Implement_optin_sharp_screenspace_reflections_for_the_defe.md

```diff
@@ -1,4 +1,31 @@
 <!-- Implement opt-in sharp screen-space reflections for the deferred renderer, with improved raymarching code. -->
 <!-- https://github.com/bevyengine/bevy/pull/13418 -->
 
-<!-- TODO -->
+{{ compare_slider(
+    path="POST_PATH",
+    left_title="No SSR",
+    left_image="no_ssr.png",
+    right_title="SSR",
+    right_image="ssr.png"
+) }}
+
+[Screen-space reflections](https://lettier.github.io/3d-game-shaders-for-beginners/screen-space-reflection.html) approximate real-time reflections by raymarching through the depth buffer and copying samples from the final rendered frame.
+Our initial implementation is relatively minimal, to provide a flexible base to build on, but is based on the production-quality [raymarching code by Tomasz Stachowiak](https://gist.github.com/h3r2tic/9c8356bdaefbe80b1a22ae0aaee192db), one of the creators of the indie darling Bevy game [Tiny Glade](https://store.steampowered.com/app/2198150/Tiny_Glade/).
+As a result, there are a few caveats to bear in mind:
+
+1. Currently, this feature is built on top of the deferred renderer and is currently only supported in that mode. Forward screen-space reflections are possible albeit uncommon (though e.g. Doom Eternal uses them); however, they require tracing from the previous frame, which would add complexity. This patch leaves the door open to implementing SSR in the forward rendering path but doesn't itself have such an implementation.
+2. Screen-space reflections aren't supported in WebGL 2, because they require sampling from the depth buffer, which `naga` can't do because of a bug (sampler2DShadow is incorrectly generated instead of sampler2D; this is the same reason why depth of field is disabled on that platform).
```

Comment:
Nit: Is there a tracking issue for this bug? Could be worth linking to it for reference (and in the off chance someone sees this and decides to tackle it themselves)?

Reply:
I can't find this after quiet a while looking. I'll add a link if it turns up.